poetry.lock

```diff
@@ -10,7 +10,7 @@ python-versions = ">=3.5"
 dev = ["cloudpickle", "coverage[toml] (>=5.0.2)", "furo", "hypothesis", "mypy (>=0.900,!=0.940)", "pre-commit", "pympler", "pytest (>=4.3.0)", "pytest-mypy-plugins", "sphinx", "sphinx-notfound-page", "zope.interface"]
 docs = ["furo", "sphinx", "sphinx-notfound-page", "zope.interface"]
 tests = ["cloudpickle", "coverage[toml] (>=5.0.2)", "hypothesis", "mypy (>=0.900,!=0.940)", "pympler", "pytest (>=4.3.0)", "pytest-mypy-plugins", "zope.interface"]
-tests-no-zope = ["cloudpickle", "coverage[toml] (>=5.0.2)", "hypothesis", "mypy (>=0.900,!=0.940)", "pympler", "pytest (>=4.3.0)", "pytest-mypy-plugins"]
```

Comment:
Since those changes are not related to the PR, should they be included in the PR? Also, you might want to update Poetry to its latest version ([1.2.2](https://github.com/python-poetry/poetry/releases/tag/1.2.2)), as extras are now normalised, to avoid a back-and-forth in the names (especially since Renovate always uses latest version of Poetry).

Reply:
@mkniewallner Thanks, missed this! I reset poetry.lock to the version in `main`.